python/tvm/contrib/debugger/debug_result.py

```diff
@@ -0,0 +1,160 @@
+"""Graph debug results dumping class."""
+import os
+import json
+import tvm
+
+GRAPH_DUMP_FILE_NAME = '_tvmdbg_graph_dump.json'
+
+class DebugResult():
+    """Graph debug data module.
+
+    Data dump module manage all the debug data formatting.
+    Output data and input graphs are formatted and the dump to files.
+    Frontend read these data and graph for visualization.
+
+    Parameters
+    ----------
+    graph_json : str
+        The graph to be deployed in json format output by nnvm graph. The graph can only contain
+        one operator(tvm_op) that points to the name of PackedFunc in the libmod.
```

Comment:
Readability: "one operator" confuse a bit. Rephrase it similar to below suggestion.

"The graph can only contain one operator(tvm_op) that points to the name of PackedFunc in the libmod."
=>
"Each operator (tvm_op) in the graph will have a one to one mapping with the symbol in libmod which is used to construct a "PackedFunc" "